Project-II-Calculator/calculator/src/components/ButtonComponents/NumberButton.js

```diff
@@ -1,2 +1,45 @@
 import React from 'react';
 import './Button.css';
+
+const Buttons = () => {
+  return (
+    <div className="cal-wrap">
+      <input type="text" >
+        
+      </div>
+
+      <div className="row">
```

Comment:
*props* is undefined. You may need to pass down *buttonStyle* from the App component where *NumberButton* is rendered.